<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>303</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>301</width>
    <height>303</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>301</width>
    <height>303</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Nastavení hry</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>Bílé figurky</string>
   </property>
   <widget class="QComboBox" name="comboBoxWhiteDifficulty">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Obtížnost umělé inteligence bílého hráče.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <item>
     <property name="text">
      <string>Začátečník</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pokročilý</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mistr</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Velmistr (trvá dlouho)</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxWhitePlayerType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Typ hráče figurek. Pokud zvolíte jako typ hráče počítač, tak tah tohoto hráče zahraje umělá inteligence.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <item>
     <property name="text">
      <string>Člověk</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Počítač</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Obtížnost:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Druh hráče:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>281</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>Černé figurky</string>
   </property>
   <widget class="QComboBox" name="comboBoxBlackDifficulty">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Obtížnost umělé inteligence černého hráče.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <item>
     <property name="text">
      <string>Začátečník</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pokročilý</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mistr</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Velmistr (trvá dlouho)</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxBlackPlayerType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>Typ hráče figurek. Pokud zvolíte jako typ hráče počítač, tak tah tohoto hráče zahraje umělá inteligence.</string>
    </property>
    <item>
     <property name="text">
      <string>Člověk</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Počítač</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Obtížnost:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Druh hráče:</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Zrušit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonApply">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Použít</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
